charts/kueue/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Default values for kueue.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Enable each function, like kustomize https://github.com/kubernetes-sigs/kueue/blob/main/config/default/kustomization.yaml
+enablePrometheus: false
```

Comment:
From my experiences, maybe this is better for future expansion? But I'm also ok with current implementation, we can evolve gradually.
```
prometheus:
    enabled: false
    // future configurations
```

Reply:
I think we can put enableWebhook and webhookService together in this way?

Reply:
Yeah, perhaps this would be better for future expansion. I'll consider it.
Thanks for your feedback!

Reply:
Thanks for your comment!
In helm, flat should be favored over nested. https://helm.sh/docs/chart_best_practices/values/
So I continue using enablePrometheus instead of your suggestion.


> In most cases, flat should be favored over nested. The reason for this is that it is simpler for template developers and users.